index.html

```diff
@@ -28,10 +28,159 @@ <h1>Muhammad Nisar Ali</h1>
+                        <li class="lang-used">JavaScript</li>
+                    </ul>
+
+                    <button class="proj-link" type="button"> See this project <img src="./Images/ic_arrow_right.svg" alt="Click and proced to check the project"></button>
+                </div>
+            </article>
+
+            <article class="proj-container last-container">
+                <img class="proj-img" src="./Images/Rectangle 21 (5).svg" alt="">
+
+                <div class="proj-description">
+                    <h3 class="proj-name">Project name goes here</h3>
+
+                    <ul class="languages">
+                        <li class="lang-used">HTML/CSS</li>
+                        <li class="lang-used">Ruby on Rails</li>
+                        <li class="lang-used">JavaScript</li>
+                    </ul>
+
+                    <button class="proj-link" type="button"> See this project <img src="./Images/ic_arrow_right.svg" alt="Click and proced to check the project"></button>
+                </div>
+            </article>
+        </section>
+
+        <section class="about-section">
+            <article class="introduction">
+                <h2 class="about"> About </h2>
+
+                <div class="my-intro">
+                    <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod </p>
+
+                    <button class="cv-btn" type="button">Get my CV <img src="./Images/ic_download.svg" alt="downlod my CV"></button>
+                </div>
+            </article>
+            
+            <article class="knowledge">
+            <div class="language">
+                <h3 class="skill-cat">Languages</h3>
+                <ul class="skill-cat-list">
+                    <li class="li">JavaScript(ES5/ES6)</li>
+                    <li class="li">Ruby</li>
+                    <li class="li">HTML</li>
+                    <li class="li">CSS</li>
+                </ul>
+            </div>
+
+            <div class="frameworks">
+                <h3 class="skill-cat">Frameworks</h3>
+                <ul class="skill-cat-list">
+                    <li class="li">Bootstrap</li>
+                    <li class="li">Ruby on rails</li>
+                    <li class="li">RSpec</li>
+                    <li class="li">Capibara</li>
+                    <li class="li">Selenium</li>
+                </ul>
+            </div>
+
+            <div class="skills-tool">
+                <h3 class="skill-cat">Skills</h3>
+                <ul class="skill-cat-list">
+                    <li class="li">Codekit</li>
+                    <li class="li">Codepen</li>
+                    <li class="li">Github</li>
+                    <li class="li">Gitlab</li>
+                    <li class="li">Terminal</li>
+                </ul>
+            </div>
```

Comment:
Please make sure The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` tag
example: 

```
<ul class="skills">
      <li class="skill-card">
        <div class="skill-title"><div class="icon rectangle"></div> Language</div>
        <ul>
          <li>Javascript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>Css</li>
        </ul>
      </li>
      <li class="skill-card">
        <div class="skill-title"><div class="icon square"></div> Frameworks</div>
        <ul>
          <li>Bootstrap</li>
          <li>Ruby on Rails</li>
          <li>RSPec</li>
          <li>CapyBara</li>
          <li>Selenium</li>
        </ul>
      </
      <li class="skill-card">
        <div class="skill-title"><div class="icon circle"></div> Skills</div>
        <ul>
          <li>Codekit</li>
          <li>Github</li>
          <li>Codepen</li>
          <li>Gitlab</li>
          <li>Terminal</li>
        </ul>
      </li>
    </ul>
```

Reply:
Thanks for your review. I have made changes as per the project requirements and your suggestion.